CHANGELOG.md

```diff
@@ -2,6 +2,33 @@
 
 Records breaking changes from major version bumps
 
+## 11.0.0
+
+PR: [#195](https://github.com/alphagov/digitalmarketplace-utils/pull/195)
+
+### What changed
+
+1. Moved the creation of the `manager` instance into `init_manager`.
+
+### Example app change
+
+#### In application's `__init__.py`
+
+Old
+```python
+from flask.ext.script import Manager
+
+manager = Manager(application)
+init_manager(manager, 5003, ['./app/content/frameworks'])
```

Comment:
This is not correct, it should be the same as the new, the only difference is that the manager is returned.